webofneeds/won-docker/README.md

```diff
@@ -13,45 +14,39 @@ the webofneeds applications as docker containers. There are images for the follo
 - gencert: a tool that can generate certificates for wonnode and owner application
 - bots: bots are used to test the communication between the servers in the application
 
-For deployment of these components we use [docker-compose](https://docs.docker.com/compose/). It is possible to deploy
-the components on multiple servers. For straight forward deployment however we have provided one docker-compose file
-which can be used to deploy all components on one local server.
+### Steps
+
+1. Download and install docker (https://www.docker.com)
+2. Download and install docker-compose (https://docs.docker.com/compose/)
+3. Download and run one of the following scripts:
+   - Bash: [deploy_bash.sh](/webofneeds/won-docker/deploy/local_image/deploy_bash.sh)
+   - PowerShell: [deploy_powershell.ps1](/webofneeds/won-docker/deploy/local_image/deploy_powershell.ps1)
+4. After Docker downloading and starting the components you can access the owner and the node here:
+   - owner: [https://10.0.75.1:8082/owner](https://10.0.75.1:8082/owner)
+   - wonnode: [https://10.0.75.1:8889/won](https://10.0.75.1:8889/won)
 
-### How to deploy and run all webofneeds components on a single server
+### Troubleshooting based on steps
 
-**We recommend using the quick guide [tutorial](/documentation/how-to-won/docker.md)**
+**Step 1:** Docker is to slow: Make sure after installing Docker Desktop, that it has enough resources on your machine `Docker Desktop` -> `Settings` -> `Advanced`
```

Comment:
As step 1 is basically "Install Docker", organising troubleshooting info by steps here may not be useful.